tests/Maker/AdminMakerTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Maker;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Sonata\AdminBundle\Maker\AdminMaker;
+use Sonata\AdminBundle\Model\ModelManagerInterface;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\FileManager;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\StreamOutput;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * @author Gaurav Singh Faujdar <faujdar@gmail.com>
+ */
+class AdminMakerTest extends TestCase
+{
+    private $projectDirectory;
+    private $modelManagers = [];
+
+    private $input;
+    private $output;
+    private $io;
+    private $generator;
+    private $servicesFile;
+
+    protected function setup()
+    {
+        if (5 == PHP_MAJOR_VERSION && !class_exists('Symfony\Component\Console\CommandLoader\CommandLoaderInterface')) {
```

Comment:
I think the problem is here, it should be `||` cc @OskarStark

Reply:
Could you please just edit the file via ui and see if it’s working?